problem-29/src/app/component/background-view/background-view.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+import {NgClass} from "@angular/common";
+
+@Component({
+  selector: 'app-background-view',
+  standalone: true,
+  imports: [
+    NgClass
+  ],
+  template:
+    `
+    <div class="main-container">
+      <!--          TODO: Add dynamic styling and mouse events on div below-->
+      <div class="container-height">
+            <p> Link style sheet to add border and background!</p>
+        </div>
+    </div>
+  `,  styleUrl: './background-view.component.scss'
+})
+export class BackgroundViewComponent {
+
+  showBlueBackground: boolean = false;
```

Comment:
add a button that toggles the property value

Reply:
Resolved - added a button that manually toggles the value